File: Neetcode/TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i, n in enumerate(nums):
            if n <= abs(target) and ((target - n) in nums):
                if nums.index(target - n) != i:
                    return [i, nums.index(target - n)]

File: Neetcode/TwoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int comp;
        unordered_map<int, int> map;
        for (int i = 0; i < nums.size(); i++){
            comp = target - nums[i];
            if (map.find(comp) != map.end()){
                return {map[comp], i};
            }
            map[nums[i]] = i;
        }
        return {};
    }
};
